### PR TITLE
Simplify Windows download entries

### PR DESCRIPTION
**Pull request by zooba**
The main aim of this is to reduce the number of items shown for Windows items, as well as to bring them more in line with how people are used to seeing them. Specifically:

* remove entries for pre-3.5 releases from the script completely
* change "x86" and "x86-64" to "(32-bit)" and "(64-bit)" (which matches the installed Start menu entries)
* remove the webinstall entries (not widely used)

There doesn't seem to be any easy way to make these changes not apply to 3.8 and 3.9, but I don't really mind if those are updated as well.

I also added the `print_function` future and put parentheses everywhere so I didn't have to install Python 2 just to double-check my syntax.

### CODE REVIEW

**Comment by zooba** [comment]
Ping @ambv @pablogsal

File: add-to-pydotorg.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 Georg Brandl, March 2014.
 """
+from __future__ import print_function
```
Comment (pablogsal) [inline add-to-pydotorg.py:25]: Oh boy, I think I would never see you again in this GitHub org :P

**Review by pablogsal — APPROVED** [review]
LGTM

File: add-to-pydotorg.py (hunk 1 above)
Comment (ned-deily) [reply add-to-pydotorg.py:25]: It looks like the download server, where RMs run the script, currently does have an Ubuntu 3.6.9 installed, so we could consider making the big leap to running under Python 3 :)

**Review by ned-deily — APPROVED** [review]

File: add-to-pydotorg.py (hunk 1 above)
Comment (zooba) [reply add-to-pydotorg.py:25]: I could have left it as only valid syntax for Python 2.x if you liked :)

**Comment by zooba** [comment]
Just want to make sure @ambv sees this before it goes in, in case it becomes a surprise for the next 3.8/3.9 releases.

**Pull request merged**